pkg/util/escape.go

```diff
@@ -20,6 +20,19 @@ import (
 	"strings"
 )
 
+// EscapeQualifiedPluginName converts a plugin name in the format
+// vendor/pluginname into a proper ondisk vendor~pluginname plugin directory
+// format.
+func EscapePluginName(in string) string {
```

Comment:
Why not use the existing `EscapeQualifiedNameForDisk(...)` and `UnescapeQualifiedNameForDisk(...)` methods?